includes/TypesForSolids/GasSolids.js

```diff
@@ -0,0 +1,77 @@
+if (!RFEM) {
+	throw new Error("This script is only for RFEM, it works with solids.");
+}
+
+/*
+this.gas_solid.temperature = temperature (doesn't work??)
+*/
+
+/**
+* Creates gas solid
+* @class
+* @constructor
+* @param	{Number}	no			Index of gas solid
+* @param	{Array}		solid_list	List of solid indexes
+* @param	{String}	comment		Comment, can be undefined
+* @param	{Object}	params  	Gas solid's parameters, can be undefined
+* @returns	Created gas solid
+*/
+function GasSolid (no,
+	solid_list,
+	comment,
+	params) {
```

Comment:
We have Solid.prototype.Gas which needs to be fixed. So that it has parameters (no, boundary_surfaces, material, gassSolidNo, comment, params)

Reply:
I noticed that no examples exist for solids, am I right?